### PR TITLE
Applicant dashboard changes

### PR DESCRIPTION
**Pull request by prekshavijay**
## Description

User specific applications and specific course type applications

### Changes

List the changes done to fix a bug or introducing a new feature.

## How to test

Describe the steps required to test the changes proposed in the pull request.

Please consider using the closing keyword if the pull request is proposed to
fix an issue already created in the repository
(https://help.github.com/articles/closing-issues-using-keywords/)

### CODE REVIEW

File: apps/wrapper/applicant-portal/.env
Comment (amit-s19) [inline apps/wrapper/applicant-portal/.env:1]: Can we please stop committing envs? 🥹

File: apps/wrapper/web-portal/src/api/apiUrl.js
Comment (amit-s19) [inline apps/wrapper/web-portal/src/api/apiUrl.js:1]: There is merge conflict headers still present here. Please remove them.

**Review by amit-s19 — APPROVED** [review]
Approving with comments

File: apps/wrapper/web-portal/src/api/apiUrl.js
Comment (prekshavijay) [reply apps/wrapper/web-portal/src/api/apiUrl.js:1]: It's done. Please check.

**Pull request merged**